components/event-bus/internal/knative/subscription/controller/subscription/controller.go

```diff
@@ -24,13 +24,13 @@ const (
 
 // ProvideController returns a Controller that represents the subscription controller. It
```

Comment:
Kindly correct the docstring after changing the function signature

Reply:
Fixed